changelog/unreleased/enhancement-resharing

```diff
@@ -0,0 +1,6 @@
+Enhancement: Re-sharing with oCIS
+
+We've implemented re-sharing of resources when using oCIS as backend.
```

Comment:
Why the two changelog items, both about adding re-sharing for ocis to web? Can we delete this changelog item here?